File: Antlr/Ejemplo2_Calc/Calc.java
LabeledExprLexer lexer = new LabeledExprLexer (input);
CommonTokenStream tokens = new CommonTokenStream(lexer);
LabeledExprParser parser = new LabeledExprParser(tokens);
ParseTree tree = parser.prog(); // Analiza la expresión
EvalVisitor eval = new EvalVisitor();
eval.visit(tree);

File: Antlr/Ejercicio Py/calc.py
import sys
from antlr4 import *
from ExprLexer import ExprLexer
from ExprParser import ExprParser
from EvalVisitor import EvalVisitor

def main():
    input_stream = FileStream(sys.argv[1]) if len(sys.argv) > 1 else InputStream(sys.stdin.read())
    lexer = ExprLexer(input_stream)
    tokens = CommonTokenStream(lexer)
    parser = ExprParser(tokens)
    tree = parser.prog()

    visitor = EvalVisitor()
    visitor.visit(tree)

if __name__ == '__main__':
    main()
